src/modules/common/data/Base.ts

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+
+    The base class that related to the block.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+export class Base
+{
+    protected checkValidity (json: any)
```

Comment:
I just saw it.
A description of `checkValidity` is required.
with a description of the input to the parameter.